opcodes/default_path.md

```diff
@@ -3,10 +3,17 @@ layout: "sfz/opcode"
 lang: "en"
 opcode_name: "default_path"
 ---
-Used under the ‹[control](/headers/control)› header.
+Used under the ‹[control](/headers/control)› header. Changes the root path for
```

Comment:
Be aware that here is inserted the `syntax.yml`s `short_description: "Relative path for samples."`, so this maybe should be changed if necessary in the yaml file to avoid redundant information.

Reply:
Right, I amended the description here and reworded the syntax.yml file.

Reply:
Thanks, maybe splitting the documentation keeping the short description in the DB was not a great idea, the main benefit it does is that you don't need to write a anything on the md file to have the minimal information, but if it creates confusion to contributors might be not that useful.

Reply:
I think it just needs to be known; maybe a short "How to contribute"? If only because creating pull requests sometimes is not that intuitive for the layman and this website may not be used only by programmers ;-)

Reply:
That is what I tried to do by writing <https://sfzformat.com/docs/website-development-reference>, maybe it needs some update and some more visible link from the navigation menus than a hidden item in a "Docs" menu.

Reply:
Yeah I have to admit I did not look too deeply for it but that's my bad ;-)